src/SFA.DAS.QnA.Application.UnitTests/CommandsTests/CreateSnapshotTests/CreateSnapshotTestBase.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NSubstitute;
+using NUnit.Framework;
+using SFA.DAS.QnA.Api.Types.Page;
+using SFA.DAS.QnA.Data;
+using SFA.DAS.QnA.Data.Entities;
+using Microsoft.Extensions.Options;
+using SFA.DAS.QnA.Configuration.Config;
+using SFA.DAS.QnA.Application.Commands.CreateSnapshot;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Storage.Blob;
+using SFA.DAS.QnA.Application.Commands.Files;
+using System.Text;
+using System.IO;
+
+namespace SFA.DAS.QnA.Application.UnitTests.CommandsTests.CreateSnapshotTests
+{
+    [TestFixture]
```

Comment:
I think this needs to move into an integration tests project, it won't be able to run on Azure Devops

Reply:
Fixed it for now using PreProcessor directives. Another option would be to mock it